requirements/requirements.txt

```diff
@@ -1,7 +1,7 @@
 mycroft-messagebus-client~=0.9.1,!=0.9.2,!=0.9.3
 inflection~=0.3
 pexpect~=4.8
-requests~=2.25
+requests>=2.26.0
```

Comment:
Consider `~=2.26` in case requests `3.0.0` is released in the future with breaking changes